docs/oadp_cheat_sheet.md

```
@@ -61,66 +61,67 @@ oc explain dataprotectionapplications.oadp.openshift.io
-velero backup create $backup_name --exclude-namespaces velero,default
+oc oadp backup create $backup_name --exclude-namespaces velero,default
 ```
 
 #### Create a backup based on a schedule named daily-backup.
 ```
-velero backup create --from-schedule daily-backup
+oc oadp backup create --from-schedule daily-backup
 ```
 
 #### View the YAML for a backup that doesn't snapshot volumes, without sending it to the server.
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)